website/docs/how-to/how-to-create-service-accounts.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: How to create service accounts
+---
+
+:::info availability
+
+Service accounts is an upcoming enterprise feature, and is scheduled to become available in one of the next few releases.
```

Comment:
This phrase is a bit vague, I'm not sure how we handle these but I'd do a couple of things:
1. Add a date of when this message is posted
2. If we're not sure on when it's going to be available, we can say that the release date is yet to be defined

```suggestion
**January, 2023:** Service accounts is an upcoming enterprise feature. The release date is yet to be defined.
```

Reply:
Thanks @gastonfournier - I think you have a point here. This was originally adapted from https://docs.getunleash.io/how-to/how-to-clone-environments - That should also probably be changed (PS: Isn't environment cloning currently GA?). 

I'll raise this discussion in Slack and tackle it on a different PR if changes are needed.